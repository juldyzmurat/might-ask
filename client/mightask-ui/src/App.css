





 .App {
  width: 100vw;
  height: 100vh;
  display: flex;
  align-items: center;
  justify-content: center;
  overflow: hidden;
 }

 .App-logo{
  width: 200px;
  height: 200px;



 }

 .CircleButton {
  position: fixed;
  top: 0px; /* Adjust top value as needed */
  right: 0px; /* Adjust right value as needed */
  transform: translateY(50%);
}

/* Tab Container */
.Tabs {
  
  width: auto;
  height: auto;
  min-height: 400px;
  background: #f1b6dc;
  margin: 3.5rem auto 1.5rem;
  padding: 2rem 1rem;
  color: #E8F0F2;
  border-radius: 2rem;
  @media (max-width: 769px) {
   padding: 2rem 0;
  }
}

/* Tab Navigation */
ul.nav {
  width: 60%;
  margin: 0 auto 2rem;
  display: flex;
  align-items: center;
  justify-content: space-between;
  border: 1px solid #39A2DB;
  border-radius: 2rem;
  padding-left: 0px;
  @media (max-width: 768px) {
    width: 90%;
  }
}
ul.nav li {
  width: 50%;
  padding: 1rem;
  list-style: none;
  text-align: center;
  cursor: pointer;
  transition: all 0.7s;
  border-bottom-left-radius: 2rem;
  border-top-left-radius: 2rem;
}
ul.nav li:nth-child(2) {
  border-radius: 0;
  border-bottom-right-radius: 2rem;
  border-top-right-radius: 2rem;
}
ul.nav li:hover {
  background: rgba(50, 224, 196, 0.15);
}
ul.nav li.active {
  background: #39A2DB;
}

.FirstTab p,
.SecondTab p {
  font-size: 2rem;
  text-align: center;
}




/*Google Maps CSS*/

.sb-title {
  position: relative;
  top: -12px;
  font-family: Roboto, sans-serif;
  font-weight: 500;
}

.sb-title-icon {
  position: relative;
  top: -5px;
}

.card-container {
  display: flex;
  height: 500px;
  width: 600px;
}

.panel {
  background: white;
  width: 300px;
  padding: 20px;
  display: flex;
  flex-direction: column;
  justify-content: space-around;
}

.half-input-container {
  display: flex;
  justify-content: space-between;
}

.half-input {
  max-width: 120px;
}

.map {
  width: 300px;
}

h2 {
  margin: 0;
  font-family: Roboto, sans-serif;
}

input {
  height: 30px;
}

input {
  border: 0;
  border-bottom: 1px solid black;
  font-size: 14px;
  font-family: Roboto, sans-serif;
  font-style: normal;
  font-weight: normal;
}

input:focus::placeholder {
  color: white;
}

.button-cta {
  height: 40px;
  width: 40%;
  background: #3367d6;
  color: white;
  font-size: 15px;
  text-transform: uppercase;
  font-family: Roboto, sans-serif;
  border: 0;
  border-radius: 3px;
  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.48);
  cursor: pointer;
}


/*Google Maps CSS*/

/* CSS for NavBar class */
.NavBar {
  background-color: #2196F3; /* Set your desired background color */
  padding: 10px; /* Adjust padding as needed */
  color: white; /* Set the text color */
  display: flex;
  justify-content: space-between;
  align-items: center;
}

.NavBar a {
  text-decoration: none;
  color: white; /* Set link color */
  margin: 0 10px; /* Add some space between links */
  font-weight: bold; /* Optionally make the text bold */
}

.NavBar a:hover {
  text-decoration: underline; /* Add underline on hover */
}

/* Optionally add a box shadow for a subtle effect */
.NavBar {
  box-shadow: 0px 2px 5px rgba(0, 0, 0, 0.1);
}
