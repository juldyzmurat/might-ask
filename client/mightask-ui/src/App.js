import React from 'react';
import gif from './20cat.gif';
import './App.css';
import CircleButton from "./Components/TaskVisComponents/accounticon"; 
import TaskView from './Components/Pages/TasksViewPage';
import Navbar from './Components/NavBar/';
import { BrowserRouter as Router, Routes, Route }
    from 'react-router-dom';
import Home from './Components/Pages/';
import About from './Components/Pages/about';
import Blogs from './Components/Pages/blogs';
import SignUp from './Components/Pages/signup';
import Contact from './Components/Pages/contact';
import Google from "./Components/Login/LoginAPI";
import { GoogleOAuthProvider } from '@react-oauth/google';
import Dashboard from "./Components/Pages/DashboardPage";


// For the cat  <img src={gif} className="App-logo" alt="gif" />


function App() {


  return (
    <div className="App">
      <header className="App-header"> 
    
      <CircleButton />
        <img src={gif} className="App-logo" alt="gif" />
      </header>
        <Router>
            <Navbar />
            
            <Routes>
                <Route exact path='/' element={<Home />} />
                <Route path='/about' element={<About />} />
                <Route path='/TasksViewPage' element={<TaskView />} />
                <Route path='/DashboardPage' element={<Dashboard />} />
                <Route path='/signup' element={<SignUp />} />
            </Routes>
        </Router>
    </div>
  );
}

export default App;



// For the cat  <img src={gif} className="App-logo" alt="gif" />


