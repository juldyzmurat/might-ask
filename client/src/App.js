import React from "react";
import "./App.css";
import TaskViews from "./Components/Pages/TaskViews";
import Navbar from "./Components/NavBar/";
import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
import Home from "./Components/Pages/";
import About from "./Components/Pages/about";
import Schedule from "./Components/Pages/schedule";
import Dashboard from "./Components/Pages/DashboardPage";

function App() {
  return (
    <div className="App">
      <header className="App-header"></header>
      <Router>
        <Navbar />
        <div classname="main-box"style={{
  background: "#f1b6dc",
  margin: "3.5rem auto 1.5rem",
  padding: "2rem 1rem",
  color: "#e8f0f2",
  borderRadius: "2rem",
  paddingTop: "5%"
}}>
          <Routes>
            <Route exact path="/" element={<Home />} />
            <Route path="/about" element={<About />} />
            <Route path="/task-views" element={<TaskViews />} />
            <Route path="/DashboardPage" element={<Dashboard />} />
            <Route path="/schedule" element={<Schedule />} />
          </Routes>
        </div>
      </Router>
    </div>
  );
}

export default App;
