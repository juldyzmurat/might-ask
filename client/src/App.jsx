import React from "react";

import "./App.css";
import TaskViews from "./Components/Pages/TaskViews";
import Navbar from "./Components/NavBar/";
import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
import Home from "./Components/Pages/";
import About from "./Components/Pages/about";
import Schedule from "./Components/Pages/schedule";
import Dashboard from "./Components/Pages/DashboardPage";
import CatStats from "./Components/Pages/catstats";
import 'bootstrap/dist/css/bootstrap.min.css';
import 'bootstrap/dist/js/bootstrap.bundle.min.js';

function App() {
  return (
    <div className="App">
      <header className="App-header"></header>
      <Router>
        <Navbar />
        {/* <div className="main-box"> */}
          <Routes>
            <Route exact path="/" element={<Home />} />
            <Route path="/about" element={<About />} />
            <Route path="/task-views" element={<TaskViews />} />
            <Route path="/DashboardPage" element={<Dashboard />} />
            <Route path="/schedule" element={<Schedule />} />
            <Route path="/catstats" element={<CatStats />} />
          </Routes>
        {/* </div> */}
      </Router>
    </div>
  );
}

export default App;
