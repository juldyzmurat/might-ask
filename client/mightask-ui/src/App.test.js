import render from '@testing-library/react';
import App from './App';

it('has a routes element', () => {
  const view = render(<App />);
  expect(view).toBeTruthy();
});