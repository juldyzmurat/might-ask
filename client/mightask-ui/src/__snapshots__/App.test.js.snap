// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <div
      onClick={[Function]}
      style={
        Object {
          "borderRadius": "50%",
          "cursor": "pointer",
          "height": "100px",
          "overflow": "hidden",
          "width": "100px",
        }
      }
    >
      <img
        alt=""
        src="kitty.jpeg"
        style={
          Object {
            "height": "100%",
            "objectFit": "cover",
            "width": "100%",
          }
        }
      />
    </div>
    <img
      alt="gif"
      className="App-logo"
      src="20cat.gif"
    />
  </header>
  <div
    className="Tabs"
  >
    <ul
      className="nav"
    >
      <li
        className="active"
        onClick={[Function]}
      >
        Tab 1
      </li>
      <li
        className=""
        onClick={[Function]}
      >
        Tab 2
      </li>
    </ul>
    <div
      className="outlet"
    >
      <div
        className="FirstTab"
      >
        <p>
          First Tab!! Hurray!! List View
        </p>
        <ul
          className="MuiList-root MuiList-padding MuiList-subheader css-1hln3ey-MuiList-root"
        >
          <li />
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Monday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Tuesday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Wednesday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Thursday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Friday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Saturday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
          <li>
            <ul>
              <li
                className="MuiListSubheader-root MuiListSubheader-gutters MuiListSubheader-sticky css-uob957-MuiListSubheader-root"
              >
                Sunday
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 0
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 1
                  </span>
                </div>
              </li>
              <li
                className="MuiListItem-root MuiListItem-gutters MuiListItem-padding css-1p823my-MuiListItem-root"
                disabled={false}
              >
                <div
                  className="MuiListItemText-root css-tlelie-MuiListItemText-root"
                >
                  <span
                    className="MuiTypography-root MuiTypography-body1 MuiListItemText-primary css-10hburv-MuiTypography-root"
                  >
                    Item 2
                  </span>
                </div>
              </li>
            </ul>
          </li>
        </ul>
      </div>
    </div>
  </div>
</div>
`;
